doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -106,6 +106,8 @@ Matter
 * Disabled OpenThread shell by default in Matter over Thread samples.
 * Enabled :kconfig:option:`CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option by default, including for builds without factory data support.
   The firmware now erases all flash pages in the non-volatile storage during a factory reset, instead of just clearing Matter-related settings.
+* Added shell commands for printing and resetting the top usage or critical resources used by the Matter stack.
+  The shell commands are available when both :kconfig:option:`CHIP_LIB_SHELL` and :kconfig:option:`CHIP_STATISTICS` Kconfig options are set.
```

Comment:
I think this should be mentioned also on advanced Kconfig configuration page.

Reply:
I think you probably mean https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/docs/guides/nrfconnect_examples_cli.md as the advanced Kconfig configuration page doesn't describe shell commands in details?

Reply:
We do mention them https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/matter/getting_started/advanced_kconfigs.html#matter-shell-commands so I thought :kconfig:option:`CHIP_STATISTICS` should be mentioned there as well.

Also, do you mean in this sentence all shell commands being available with these two Kconfigs enabled or only the shell commands for printing and resetting? If only the latter, then the line should start with "These shell commands".

Reply:
Only the shell commands for printing and resetting.

Reply:
I think it would be cleaner if we extended nrfconnect_examples.md as there are other commands that are available only if we enable some extra features - for example, nfc commands are available only if NFC is enabled, and ota commands are available only when the OTA requestor is enabled. So it would be strange if we only decided to mention `CHIP_STATISTICS` in the main doc.

Reply:
Created https://github.com/nrfconnect/sdk-connectedhomeip/pull/318.